website/docs/authenticating_via_service_principal.html.markdown

```diff
@@ -113,14 +113,14 @@ $ az account list-locations
 
 ### Creating a Service Principal in the Azure Portal
 
-There's a couple of phases to create a Service Principal via [the Azure Portal](https://portal.azure.com):
+We need to do two more steps to create a Service Principal via [the Azure Portal](https://portal.azure.com):
```

Comment:
I think this'd read better as `There's two tasks needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):` or `Two tasks are needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):` - what do you think?

Reply:
That sounds good to me!